tests/cupyx_tests/scipy_tests/special_tests/test_log_softmax.py

```diff
@@ -0,0 +1,49 @@
+import pytest
+
+import numpy
+
+import cupy
+from cupy import testing
+import cupyx.scipy.special  # NOQA
+
+import scipy.special  # NOQA
+
+
+@testing.with_requires('scipy')
+class TestLogSoftmax:
+
+    @testing.for_all_dtypes()
+    @testing.numpy_cupy_array_equal(scipy_name='scp')
```

Comment:
Please use `numpy_cupy_allclose` to test floating point values.

Reply:
Supplying tolerance as following would work. (though, maybe you'll have more about dtypes)

```py
atol = {'default': 1e-6, cupy.float64: 1e-14}
rtol = {'default': 1e-6, cupy.float64: 1e-14}

@testing.for_all_dtypes()
@testing.numpy_cupy_allclose(scipy_name='scp', atol=atol, rtol=rtol)
def test_log_softmax_large_inputs(self, xp, scp, dtype):
    a = xp.arange(4, dtype=dtype)
    return scp.special.log_softmax(a)
```

Reply:
Done